commit/solvers.py

```diff
@@ -284,22 +284,23 @@ def fista( y, A, At, tol_fun, tol_x, max_iter, verbose, x0, omega, proximal) :
     """
 
     # Initialization
-    res = -y.copy()
+    
     xhat = x0.copy()
     x = np.zeros_like(xhat)
-    res += A.dot(xhat)
+    res = confidence_map * (A.dot(xhat) -y.copy()) 
```

Comment:
`confidence_map` seems an array: why not calling it instead `confidence_array` or simply `confidedence_values`? Using the term _map_ makes me think about a 3d image (e.g. ICVF map, myelin map etc)

Reply:
Agree. I was thinking to call it `confidence_vec` but I think `array` is better than `vec.`